cloud: Update monitor-built-in-alerting.md (#12980)
pingcap/docs: tidb-cloud/monitor-built-in-alerting.md

1. Log in to the [TiDB Cloud console](https://tidbcloud.com).
2. In the left navigation pane of the [**Clusters**](https://tidbcloud.com/console/clusters) page, do one of the following:

    - If you have multiple projects, switch to the target project, and then click **Admin** > **Alert Subscription**.
    - If you only have one project, click **Admin** > **Alert Subscription**.

> 3. Click **Add Subscriber**, enter your email address in the popup window and then click **Add**.

Reviewer: ```suggestion
3. Click **Add Subscriber**, enter your email address in the displayed dialog, and then click **Add**.
```

> If the condition triggering an alert does not change, the alert will send email notifications every 3 hours .

Reviewer: ```suggestion
If the condition that triggers an alert remains unchanged, the alert sends email notifications every 3 hours.
```

Reviewer: alert condition 和这篇文档后面的 “TiDB Cloud built-in alert conditions” 也是对应的

1. Log in to the [TiDB Cloud console](https://tidbcloud.com).
2. In the left navigation pane of the [**Clusters**](https://tidbcloud.com/console/clusters) page, do one of the following:

    - If you have multiple projects, switch to the target project, and then click **Admin** > **Alert Subscription**.
    - If you only have one project, click **Admin** > **Alert Subscription**.

> 3. Locate your email address and click **Unsubscribe**.

Reviewer: ```suggestion
3. Locate your email address and click **Unsubscribe**.
4. Click **Delete** to confirm the unsubscription.
```
